src/vs/editor/contrib/suggest/suggestWidget.ts

```diff
@@ -504,10 +501,17 @@ export class SuggestWidget implements IContentWidget, IVirtualDelegate<ICompleti
 	}
 
 	private _getSuggestionAriaAlertLabel(item: ICompletionItem): string {
-		if (canExpandCompletionItem(item)) {
-			return nls.localize('ariaCurrentSuggestionWithDetails', "{0}, suggestion, has details", item.suggestion.label);
+		const isSnippet = item.support === getSnippetSuggestSupport();
```

Comment:
Careful, not sure what you are aiming for but there is the snippet provider that only returns suggestions of type snippet (as you use it here). Then there are suggestions of type snippets that any provider can return. Last, a completion item provider might return a suggestion that's not of type snippet but its insert text behaves like a snippet (what emmet does). 

So, not sure what you are aiming for. Today, snippets, from snippets files, can visually be identified via the extra info pop up but not by looking at the list alone. If this is about the snippet type (the icon you see) then I would go for the type of the suggestion. Need more info for me to make a call.

Reply:
@jrieken My initial aim was to only target the snippets from the snippet provider. But, the same can be extended to snippets from providers as well. I've updated the code to look at the suggestion type.

Yes, we can still have providers that insert snippet, but dont mark it as such. That would be out of scope of this PR